tests/CommonExtractorTest.php

```diff
@@ -799,6 +800,31 @@ public function testIncrementalFetchingInvalidConfig(): void
         }
     }
 
+    public function testColumnOrdering(): void
+    {
+        $this->createAutoIncrementAndTimestampTable();
+        $config = $this->getIncrementalFetchingConfig();
+        $config['parameters']['columns'] = ['timestamp', 'id', 'name'];
+        $config['parameters']['outputTable'] = 'in.c-main.columnsCheck';
+        $result = $this->getApp($config)->run();
+        $this->assertEquals('success', $result['status']);
+        $outputManifestFile = $this->dataDir . '/out/tables/in.c-main.columnsCheck.csv.manifest';
+
+        $outputManifest = json_decode(file_get_contents($outputManifestFile), true);
+
+        // check that the manifest has the correct column ordering
+        $this->assertEquals($config['parameters']['columns'], $outputManifest['columns']);
+        // check the data
+        $expectedData = iterator_to_array(new CsvFile($this->dataDir.'/columnsOrderCheck.csv'));
+        $outputData = iterator_to_array(new CsvFile($this->dataDir.'/out/tables/in.c-main.columnsCheck.csv'));
+        foreach ($outputData as $rowNum => $line) {
+            // assert timestamp
+            $this->assertNotFalse(strtotime($line[0]));
```

Comment:
The test table will always have 2 lines of data, I should assert that by count.  I'm checking the lines because I want to assert that the first column is the timestamp data, but because its a timestamp, the value will always be different.

Reply:
but `$outputData` file is created by extractor? so it can be empty

Reply:
yes, it's created by the extractor, but if it is empty the test should fail. (I'm asserting it's not empty in my latest commit)